common/commands.mk

```diff
@@ -9,6 +9,7 @@ V_FLAGS := -prod -no-bounds-checking -prealloc -cflags "-Wno-error=implicit-func
 V_VSL_CBLAS_FLAGS := $(V_FLAGS) -d cblas
 ZIG_FLAGS := -O ReleaseFast
 
+C3_BUILD =	c3c -05 --safe=no $^
```

Comment:
Regarding the --safe flag: it's to disable bounds check, similar as it's done for other languages.

Ref.:
```
--safe=<yes|no>            - Turn safety (contracts, runtime bounds checking, null pointer checks etc) on or off.
```

Reply:
Yes, that's expected to be set, thanks for the clarification.